pkg/client/result.go

```diff
@@ -14,6 +16,31 @@ type Result struct {
 	Rows    []Row    `json:"rows"`
 }
 
+// Due to big int number limitations in javascript, numbers should be encoded
+// as strings so they could be properly loaded on the frontend.
+func (res *Result) PrepareBigints() {
+	for i, row := range res.Rows {
+		for j, col := range row {
+			if col == nil {
+				continue
+			}
+
+			switch reflect.TypeOf(col).Kind() {
+			case reflect.Int64:
+				val := col.(int64)
+				if val < -9007199254740991 || val > 9007199254740991 {
+					res.Rows[i][j] = strconv.FormatInt(col.(int64), 10)
+				}
```

Comment:
For what its worth, I don't really agree with dynamically changing the shape of the json object based on the values.

For example, consider this code:

```javascript
while (true) {
    var result = JSON.parse(getValueFromApi());
    var oneHigher = result.usuallyNumber + 1;
    saveToDb({usuallyNumber: oneHigher});
}
```

If `result.usuallyNumber` is all of a sudden a string (because your values got too big), the `+` operator in javascript ends up becoming string concatenation.

```
1
2
3
...
9007199254740989
9007199254740990
9007199254740991
90071992547409911
900719925474099111
9007199254740991111
```